mercury/mercury-library/protocol-issue-credential/src/test/scala/io/iohk/atala/mercury/protocol/anotherclasspath/UtilsCredentialSpec.scala

```diff
@@ -40,15 +40,19 @@ class UtilsCredentialSpec extends ZSuite {
       .build(
         fromDID = DidId("did:prism:test123from"),
         toDID = DidId("did:prism:test123to"),
-        credentials = Map(nameCredentialType -> credential.asJson.noSpaces.getBytes()),
+        credentials =
+          Seq(IssueCredentialIssuedFormat.Unsupported(nameCredentialType) -> credential.asJson.noSpaces.getBytes()),
```

Comment:
Possible improvement: the code has many `.asJson.noSpaces.getBytes()` entries.
It might be good to implement the general decoder/encoder for this.

Reply:
We have (this is a test for that if I'm not mistaken)